src/minimega/container.go

```diff
@@ -1359,9 +1367,21 @@ func containerPopulateCgroups(vmID, vmMemory int) error {
 		}
 	}
 
+	// vcpus
```

Comment:
can we reference something that indicates that the period/quota calculation here maps to an integer cpu unit?